cmd/argocd/commands/root.go

```diff
@@ -136,3 +93,33 @@ func NewCommand() *cobra.Command {
 
 	return command
 }
+
+// HandleCommandExecutionError processes the error returned from executing the command.
+// It handles both standard Argo CD commands and plugin commands.
+func HandleCommandExecutionError(err error, isArgocdCLI bool, o ArgoCDCLIOptions) {
+	if err != nil {
+		if isArgocdCLI && strings.Contains(err.Error(), "unknown command") {
+			// If it's an unknown command error, attempt to handle it as a plugin.
+			// Unfortunately, cobra doesn't handle this error, so we need to assume
```

Comment:
Please add the link to the Cobra PR that will introduce this error type in the comment. This way future devs can keep an eye on that.

Reply:
resolved